content/docs/user-guide/pipelines/index.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Pipelines Management in DVC
+---
+
+# What is a (DVC) pipeline?
+
+A machine learning pipeline is like an assembly line in a factory. As raw
+materials input to a factory and processed at each step to achieve a final
+product, a machine learning pipeline gets the data as an input, processes it in
+every stage and outputs a model.
+
+If you are repeating a set of commands to get to the final artifacts (models,
+results), then you already have a pipeline, albeit in a manual fashion. DVC
+allows to automate your workflow, be it comprised of a single command or many
```

Comment:
I think it's not so much about manual vs automatic. You can automate any set of commands with a bash script.

DVC codifies it into a readable, std. format + provides commands to run it (which are manual too).

Reply:
Feel free to double check with the DVC team cc @dberenbaum

Reply:
Reproduction can't work without codification first. We don't have functionality to find the relationships between stages automatically.